src/proto/Algorand.proto

```diff
@@ -7,23 +7,39 @@ message TransactionPay {
     string to_address = 1;
     uint64 fee = 2;
     uint64 amount = 3;
-    uint64 first_round = 4;
-    uint64 last_round = 5;
+}
+
+message AssetTransaction {
+    string to_address = 1;
+    uint64 fee = 2;
```

Comment:
I wonder if we need to move `fee` on `SigningInput` level, since it's used for everywhere input